lib/luadata.c

```diff
@@ -24,6 +24,7 @@
 #define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
 #include <linux/kernel.h>
 #include <linux/module.h>
+#include <linux/types.h>
```

Comment:
I don't think it's necessary as we include this on [include/stdint.h](https://github.com/luainkernel/lunatik/blob/master/include/stdint.h) header.